app/helpers/forever_style_guide/application_helper.rb

```diff
@@ -70,8 +70,13 @@ def has_ambassador?
     end
 
     #active state nav
-    def is_active?(controller_name)
-      return true if [controller.controller_name, controller.action_name].include? controller_name
+    def is_active?(controller_names, product_type)
```

Comment:
Definitely not sure this is the best refactor we can do, but food for thought: 
If we remove the arrays and deal product type in the markup, pretty sure we can simplify this down to
```
def is_active?(controller_name, product_type = nil)
      controller.controller_name.include?(controller_name) || controller.action_name.include?(controller_name) || (@product != nil && @product.product_type == product_type && !@product.name.include?('Historian'))
end
```

Which is still awkward, but feels a little bit simpler?

Reply:
I'm also perplexed by using  'controller_name' when it's not really controller name, but I also haven't come up with a better idea.